.github/workflows/tests.yml

```diff
@@ -103,14 +103,17 @@ jobs:
       - name: Cache Conda
         uses: actions/cache@6849a6489940f00c2f30c0fb92c6274307ccb58a # v4.1.2
         with:
-          path: ~/conda_pkgs_dir
+          # Use faster GNU tar for all runners
+          enableCrossOsArchive: true
+          path: D:\conda_pkgs_dir
           key: cache-${{ env.HASH }}
 
       - name: Setup Miniconda
-        uses: conda-incubator/setup-miniconda@a4260408e20b96e80095f42ff7f1a15b27dd94ca # v3.0.4
+        uses: conda-incubator/setup-miniconda@d2e6a045a86077fb6cad6f5adf368e9076ddaa8d # v3.1.0
         with:
           condarc-file: .github\condarc-${{ matrix.default-channel }}
           run-post: false  # skip post cleanup
+          pkgs-dirs: D:\conda_pkgs_dir
```

Comment:
We also want to change the installation directory so they both cache and install coexist in the same drive. Otherwise conda will have to copy files instead of hardlinking, with the associated overhead.

Reply:
```suggestion
          pkgs-dirs: D:\conda_pkgs_dir
          installation-dir: D:\conda
```

Reply:
`installation-dir` is new?

Reply:
Filed https://github.com/conda/conda-build/pull/5532 ;)